core/Pos/Binary/Class/Primitive.hs

```diff
@@ -138,27 +161,14 @@ deserializeOrFail' = deserializeOrFail . BSL.fromStrict
 putCopyBi :: Bi a => a -> Contained Cereal.Put
 putCopyBi = contain . safePut . serialize
 
+-- Should there be a nocheck version of this function too?
```

Comment:
Given that we use safecopy for acid-state (i.e. a database), probably yes. It'll make parsing 1GB event logs faster. (On the other hand, maybe we'll kill acid-state soon? No idea.)

Reply:
Let's leave it as it is. ACID-state guys will decide what should they do by themselves.